apps/extension/src/ui/apps/popup/components/TotalFiatBalance.tsx

```diff
@@ -85,29 +86,42 @@ const ANALYTICS_PAGE: AnalyticsPage = {
 const TopActions = ({ disabled }: { disabled?: boolean }) => {
   const { t } = useTranslation()
   const { open: openCopyAddressModal } = useCopyAddressModal()
+  const ownedAccounts = useAccounts("owned")
   const canBuy = useIsFeatureEnabled("BUY_CRYPTO")
 
+  const { disableActions, disabledReason } = useMemo(() => {
+    const disableActions = !ownedAccounts.length
+    const disabledReason = disableActions ? t("You don't own any accounts yet") : undefined
```

Comment:
Suggest change message to: `Add an account to send or receive funds`.

Reply:
nice, thanks!